kustomize/move2kube/README.md

```diff
@@ -0,0 +1,76 @@
+## Prerequisites
+* The manifests are generated without the `namespace` configuration
+* Manifest names reflect the `resources` section of [kustomization.yaml](./base/kustomization.yaml)
+
+A brief explanation of the system architecture is provided in the related [document](./move2kube.md).
+
+## Kustomization options
+### Image update
+Run the following to update the default images to a custom configuration
+```
+cd overlays/prod && kustomize edit set image serverless-workflow-move2kube=quay.io/orchestrator/serverless-workflow-move2kube@1234
+cd overlays/prod && kustomize edit set image serverless-workflow-m2k-kfunc=quay.io/orchestrator/serverless-workflow-m2k-func@1234
+```
+
+## Configure properties
+Edit configuration in [replace-config](./overlays/prod/replace-config.yaml), [config.properties](./overlays/prod/config.properties) and 
+[secret.properties](./overlays/prod/secret.properties) to match your environment configuration.
+
+## Deploy to the cluster
+### Deploy the production environment
+This environment applies the generated manifests with minimal customizations to:
+* Use the latest application images
+* Force the `prod` profile for the SonataFlow instance
+* Deploy by default on the `sonataflow-infra` namespace
+* Mount the configurations defined in [Configure properties](#configure-properties) as environment variables
+
+The default namespace can be customized with:
+```bash
+TARGET_NS=YOUR-NS
+cd base && kustomize edit set namespace=$TARGET_NS && cd ..
+```
+
+Once the configuration is set, apply the deployment to the configured namespace with:
+```bash
+kustomize build overlays/prod | oc apply -f -
+```
+
+### Status check
```

Comment:
right after the deployment, we have to update the move2kube instance URL as this is the one used in the notification (with the link) so we have to update it to the reachable `route`

Reply:
I just saw that second deployment step but I'd propose a simpler alternative like we did for escalation, e.g. let the deployer define the cluster host in the property MOVE2KUBE_URL that is already exposed in `config.properties`.
```
MOVE2KUBE_URL=https://move2kube.sonataflow-infra-apps.ocp-dev01.lab.eng.tlv2.redhat.com
```